docs/source/tls.rst

```diff
@@ -113,6 +115,43 @@ To find the fingerprint of a given certificate, use the following OpenSSL comman
 
 SHA256 and SHA512 fingerprints are supported.  SHA1 fingerprints are not supported due to the insecurity of the SHA1 algorithm.
 
+Skip Certificate Validation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are in a development environment and need to turn off certificate validation, you can add a `SkipReceptorNamesCheck` key-value pair in your configuration file for `tls-server`, `tls-config`, or both.
```

Comment:
there is a valid, production-level use case for having this set to True. If a user has separate certs for the TLS TCP backend connection and the control service, then having the receptor name in the TLS TCP cert is not required. If the receptor name is not present, the user must set `skipreceptornamescheck` to True for that cert definition.

Reply:
i made this block of code less ambiguous. please recheck @fosterseth !